components/resources/commands.grdp

```diff
@@ -0,0 +1,487 @@
+    strings here.
+  -->
+  <message name="IDS_IDC_BACK">
+    Back
+  </message>
+  <message name="IDS_IDC_FORWARD">
+    Forward
+  </message>
+  <message name="IDS_IDC_HOME">
+    Home
+  </message>
+  <message name="IDS_IDC_STOP">
+    Stop
+  </message>
+  <message name="IDS_IDC_RELOAD_BYPASSING_CACHE">
+    Reload Bypassing Cache
+  </message>
+  <message name="IDS_IDC_RELOAD_CLEARING_CACHE">
+    Reload Clearing Cache
+  </message>
+  <message name="IDS_IDC_CLOSE_WINDOW">
+    Close Window
+  </message>
+  <message name="IDS_IDC_ALWAYS_ON_TOP">
+    Toggle Always On Top
+  </message>
+  <message name="IDS_IDC_CLOSE_TAB">
+    Close Tab
+  </message>
+  <message name="IDS_IDC_SELECT_NEXT_TAB">
+    Next Tab
+  </message>
+  <message name="IDS_IDC_SELECT_PREVIOUS_TAB">
+    Previous Tab
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_0">
+    Select Tab 0
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_1">
+    Select Tab 1
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_2">
+    Select Tab 2
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_3">
+    Select Tab 3
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_4">
+    Select Tab 4
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_5">
+    Select Tab 5
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_6">
+    Select Tab 6
+  </message>
+  <message name="IDS_IDC_SELECT_TAB_7">
+    Select Tab 7
+  </message>
+  <message name="IDS_IDC_SELECT_LAST_TAB">
+    Select Last Tab
+  </message>
+  <message name="IDS_IDC_DUPLICATE_TAB">
+    Duplicate Tab
+  </message>
+  <message name="IDS_IDC_MOVE_TAB_NEXT">
+    Move Tab Next
+  </message>
+  <message name="IDS_IDC_MOVE_TAB_PREVIOUS">
+    Move Tab Previous
+  </message>
+  <message name="IDS_IDC_SEARCH">
+    Search
+  </message>
+  <message name="IDS_IDC_WINDOW_MENU">
+    Window Menu
+  </message>
+  <message name="IDS_IDC_MINIMIZE_WINDOW">
+    Minimize Window
+  </message>
+  <message name="IDS_IDC_MAXIMIZE_WINDOW">
+    Maximize Window
+  </message>
+  <message name="IDS_IDC_ALL_WINDOWS_FRONT">
+    Show all Windows
+  </message>
+  <message name="IDS_IDC_TOGGLE_MULTITASK_MENU">
+    Show Multitask Menu
+  </message>
+  <message name="IDS_IDC_USE_SYSTEM_TITLE_BAR">
+    Toggle System Title Bar
+  </message>
+  <message name="IDS_IDC_OPEN_IN_PWA_WINDOW">
+    Open In PWA Window
+  </message>
+  <message name="IDS_IDC_WEB_APP_SETTINGS">
+    Web App Settings
+  </message>
+  <message name="IDS_IDC_WEB_APP_MENU_APP_INFO">
+    Web App Info
+  </message>
+  <message name="IDS_IDC_VISIT_DESKTOP_OF_LRU_USER_2">
+    Visit Desktop Of LRU User 2
+  </message>
+  <message name="IDS_IDC_VISIT_DESKTOP_OF_LRU_USER_3">
+    Visit Desktop Of LRU User 3
+  </message>
+  <message name="IDS_IDC_VISIT_DESKTOP_OF_LRU_USER_4">
+    Visit Desktop Of LRU User 4
+  </message>
+  <message name="IDS_IDC_VISIT_DESKTOP_OF_LRU_USER_5">
+    Visit Desktop Of LRU User 5
+  </message>
```

Comment:
In upstream code the corresponding commands are behind the `#if BUILDFLAG(IS_CHROMEOS_ASH)` flag. It would be ideal if we didn't bring in the commands we would never use, so we don't translate unnecessarily.